### ISSUE TITLE
default to regular pangeo logo

### ISSUE THREAD

**Issue by rabernat** [pangeo-forge/pangeo-sphinx-book-theme#2]
Since we may want to use this theme throughout Pangeo (not just Pangeo Forge), you may want to include the standard logo as a default: https://github.com/pangeo-data/branding/blob/master/logo/v_small_pangeo_logo.png

**Comment by cisaacstern** [comment]
Good call. In the released version `0.1.1`, there is now **no default** logo. Users can set a logo of their choosing by adding the image file to a static directory, and declaring its location and name in their `conf.py`, e.g.:

```python
html_logo = "_static/pangeo-forge-logo-blue.png"
html_static_path = ["_static"]
```

In a future release we can set the Pangeo image above as default if desired. Do you envision any sites aside from https://pangeo.io/ will want to use that default image?

**Comment by rabernat** [comment]
We use this theme on a lot of websites in the pangeo ecosystem, e.g.

- https://pangeo.io/
- https://xgcm.readthedocs.io/en/latest/
- https://pangeo-data.github.io/pangeo-cmip6-cloud/
- http://mldata.pangeo.io/

So I think having the default logo as part of the theme would be nice but not necessary or urgent in any way.

Thanks for your quick work on this task btw! 👏
